vms/event/forms.py

```diff
@@ -20,8 +20,9 @@ class Meta:
             ]
 
     def clean(self):
-        start_date = self.cleaned_data.get('start_date')
-        end_date = self.cleaned_data.get('end_date')
+        cleaned_data = super(EventForm, self).clean()
+        start_date = cleaned_data.get('start_date')
+        end_date = cleaned_data.get('end_date')
 
```

Comment:
@smarshy I did that so that the normal validation done by django takes place. by adding the `unique` constraint and also calling the builtin validate function, the names being unique will be handled by django itself and no more code required! :D